src/core/ddsc/src/dds_entity.c

```diff
@@ -1020,10 +1025,12 @@ static void pushdown_listener (dds_entity *e)
   ddsrt_mutex_unlock (&e->m_mutex);
 }
 
+
 dds_return_t dds_set_listener (dds_entity_t entity, const dds_listener_t *listener)
```

Comment:
Makes sense. Which documentation files would be impacted by this?

Reply:
Only `dds_set_listener` in `dds.h`. It just needs a bit of text like:

> `dds_set_listener` will invoke any listeners for which the corresponding status flag is set. It may cause spurious invocations, including multiple invocations for one listener. For most cases this is unlikely, but for the `DATA_ON_READERS` listeners it is quite likely, though not certain.

At least that would work for me, but YMMV. Suggestions always welcome 🙂